lhctl/cmd/principal.go

```diff
@@ -87,4 +87,5 @@ var (
 func init() {
 	putCmd.AddCommand(putPrincipalCmd)
 	putPrincipalCmd.Flags().String("acl", "", "ACLs")
+	putPrincipalCmd.Flags().String("tenantId", "", "Tenant associated with the principal")
```

Comment:
I thought that the `Principal` had a `map<string, ServerACLs> per_tenant_acls` which allows different ACL's per tenant. Is this possible to create using `lhctl` or is that not posible?